CHANGELOG.md

```diff
@@ -10,9 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Answer responses for Forms
 - Language_code field on imports and exports
+- whatsapp body hidden characters validation
```

Comment:
I wouldn't call this validation, I think "removal" is a better word here, since we're automatically removing them from the whatsapp body